src/ReadVTK.jl

```diff
@@ -411,8 +411,14 @@ function get_data_section(vtk_file::VTKFile, section)
   return VTKData(names, data_arrays, vtk_file)
 end
 
-# Retrieve FieldData section from the VTK file
-function get_field(vtk_file::VTKFile)
+"""
+  get_field_data(vtk_file::VTKFile)
```

Comment:
```suggestion
    get_field_data(vtk_file::VTKFile)
```
4 spaces are needed (or typically used) to render this as code